app/src/App.js

```diff
@@ -1,12 +1,11 @@
 import React, { Component } from 'react';
+import Home from './Home';
 
 class App extends Component {
   render() {
     return (
```

Comment:
in your text editor can you make the setting change that converts tabs to spaces

Reply:
2 or 4?